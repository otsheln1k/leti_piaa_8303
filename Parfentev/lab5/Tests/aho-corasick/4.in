CCGTGGTGT
3
GT
GGTG
CG
