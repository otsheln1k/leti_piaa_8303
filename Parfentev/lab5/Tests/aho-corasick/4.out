2 3
3 1
5 2
6 1
8 1
